Wire-iOS/Sources/Analytics/AnalyticsCallingTracker.swift

```diff
@@ -120,10 +120,24 @@ extension AnalyticsCallingTracker: WireCallCenterCallStateObserver {
             }
 
             callParticipantObserverToken = WireCallCenterV3.addCallParticipantObserver(observer: self, for: conversation, userSession: userSession)
-
-        case .terminating(reason: let reason):
+        case .terminating(let reason):
             if let callInfo = callInfos[conversationId] {
-                analytics.tag(callEvent: .ended(reason: reason.analyticsValue), in: conversation, callInfo: callInfo)
+                let video = conversation.voiceChannel?.isVideoCall ?? false
```

Comment:
suggests a clearer name
```suggestion
                let isVideoCall = conversation.voiceChannel?.isVideoCall ?? false
```